### PR TITLE
gh-99276 - Updated Doc/faq/general.rst

### PR DESCRIPTION
**Pull request by workingpayload**
* Issue: gh-99276

### CODE REVIEW

**Review by hugovk — CHANGES_REQUESTED** [review]
Thanks for the PR, but:

* this is an article not a book, so doesn't belong in the books section
* this article is already mentioned in the previous section about articles: https://docs.python.org/3/faq/general.html#are-there-any-published-articles-about-python-that-i-can-reference

Issue gh-99276 was about adding the https://ir.cwi.nl/pub/18204 article link to that articles section:

* https://docs.python.org/3/faq/general.html#are-there-any-published-articles-about-python-that-i-can-reference

Please could you update the PR?

**Comment by workingpayload** [comment]
> Thanks for the PR, but:
> 
> * this is an article not a book, so doesn't belong in the books section
> * this article is already mentioned in the previous section about articles: https://docs.python.org/3/faq/general.html#are-there-any-published-articles-about-python-that-i-can-reference
> 
> Issue [gh-99276](https://github.com/python/cpython/issues/99276) was about adding the https://ir.cwi.nl/pub/18204 article link to that articles section:
> 
> * https://docs.python.org/3/faq/general.html#are-there-any-published-articles-about-python-that-i-can-reference
> 
> Please could you update the PR?

Yeah Sure!!

**Comment by workingpayload** [comment]
I have made the requested changes; please review again

**Comment by hugovk** [comment]
Rather than adding that extra line, shall we add an inline link?

For example, to add the link around "very first article":

```diff
-The very first article about Python was written in 1991 and is now quite
-outdated.
+The `very first article <https://ir.cwi.nl/pub/18204>` about Python was 
+written in 1991 and is now quite outdated.
```

**Comment by workingpayload** [comment]
> Rather than adding that extra line, shall we add an inline link?
> 
> For example, to add the link around "very first article":
> 
> ```diff
> -The very first article about Python was written in 1991 and is now quite
> -outdated.
> +The `very first article <https://ir.cwi.nl/pub/18204>` about Python was 
> +written in 1991 and is now quite outdated.
> ```
> 
> 

Yeah sure!!
I'll do that.
Thank you for helping me out.

**Comment by workingpayload** [comment]
I have made the requested changes; please review again

File: Doc/faq/general.rst [hunk 1]
```diff
@@ -248,14 +248,15 @@ Are there any published articles about Python that I can reference?
...
-outdated.
+The very first article <https://ir.cwi.nl/pub/18204> about Python was
+written in 1991 and is now quite outdated.
 
     Guido van Rossum and Jelke de Boer, "Interactively Testing Remote Servers
     Using the Python Programming Language", CWI Quarterly, Volume 4, Issue 4
     (December 1991), Amsterdam, pp 283--303.
 
 
+
```
Comment (hugovk) [inline Doc/faq/general.rst]: Don't add a blank line:

```suggestion
```

File: Doc/faq/general.rst [hunk 2]
```diff
@@ -248,14 +248,15 @@ Are there any published articles about Python that I can reference?
 
 It's probably best to cite your favorite book about Python.
 
-The very first article about Python was written in 1991 and is now quite
-outdated.
+The very first article <https://ir.cwi.nl/pub/18204> about Python was
```
Comment (hugovk) [inline Doc/faq/general.rst]: Like this to make an inline link:
```suggestion
The `very first article <https://ir.cwi.nl/pub/18204>_` about Python was
```

(Docs: https://devguide.python.org/documentation/markup/#hyperlinks)

**Comment by workingpayload** [comment]
I have made the requested changes; please review again

File: Doc/faq/general.rst [hunk 3]
```diff
@@ -248,7 +248,7 @@ Are there any published articles about Python that I can reference?
 
 It's probably best to cite your favorite book about Python.
 
-The very first article about Python was written in 1991 and is now quite
+The `very first article <https://ir.cwi.nl/pub/18204>`_ about Python was written in 1991 and is now quite
 outdated.
```
Comment (hugovk) [inline Doc/faq/general.rst]: Let's stick to max 80 chars:

https://devguide.python.org/documentation/style-guide/#use-of-whitespace

```suggestion
The `very first article <https://ir.cwi.nl/pub/18204>`_ about Python was
written in 1991 and is now quite outdated.
```

A couple of tips:

* You can click "Commit suggestion" to directly apply this to the PR
* No need to click the "Update branch" button every time, it just creates unnecessary builds

Comment (workingpayload) [reply Doc/faq/general.rst]: okay!!
Thank you so much for helping me out.
This is my contribution that's why you are having so much trouble working with me.
I apologise for that.

Comment (hugovk) [reply Doc/faq/general.rst]: No need to apologise, thank you for the updates!

**Review by hugovk — APPROVED** [review]
Thanks!

**Pull request merged**